### ISSUE TITLE
Character creation/selection

### ISSUE THREAD

**Issue by bdrogja** [EveronLife/EveronLife#3]
**Is your feature request related to a problem? Please describe.**
You need obviosly a starting point to get to the server.

**Describe the solution you'd like**
The goal is to have a pre-first spawn screen that enables the player to customize his name and appearance. 
This should be stored persistently.

**Comment by MrPentha** [comment]
I will take responsibility for the Character creation/selection.

**Comment by bundoggity** [comment]
https://user-images.githubusercontent.com/61765983/173491487-a338bb52-2adb-4d95-8ef1-4f2313d57e07.mp4

Progress of the UI so far. Very barebones but will be spending more time figuring out how to require a selection of character before allowing the player to spawn.

**Comment by MrPentha** [comment]
The UI for the spawn selection is not finished yet, but it works already. The Spawns are configurable through a config file (displayName, position, rotation). I dont want to share any UI yet publicly, but if you want to work together with me,we can talk on DC.

**Comment by bundoggity** [comment]
Yeah that would be great! It would be nice to have someone who can give me criticism on my work. The language is new to me so any guidance would be nice. Are you in the EL discord?

**Comment by MrPentha** [comment]
Yes i am. Just DM MrPentha(Tim)#8462

**Comment by MrPentha** [comment]
Hello Team,

I don't have the time or energy to edit this feature right now. I very much regret that and will share the current status with you. Anyone who wants to continue working on this stand is welcome to do so. If anyone has any questions, they are welcome to contact me. The code isn't refactored and the UI isn't overhauled either, because I waited for the persistence system before making changes that I would have to change again later.

https://github.com/MrPentha/EveronLife/commit/dafd0f9c0cc11c5b58a7ff7cb0037d5503d0673a
